sdks/python/apache_beam/io/gcp/internal/clients/bigquery/bigquery_v2_messages.py

```diff
@@ -2194,6 +2643,7 @@ class JobConfigurationQuery(_messages.Message):
     clustering: [Beta] Clustering specification for the destination table.
       Must be specified with time-based partitioning, data in the table will
```

Comment:
@pabloem This PR doesn't seem to fix https://github.com/apache/beam/pull/12998.

It's still showing wrong info:
https://github.com/ryanyuan/beam/blob/master/sdks/python/apache_beam/io/gcp/internal/clients/bigquery/bigquery_v2_messages.py#L2031
https://github.com/ryanyuan/beam/blob/master/sdks/python/apache_beam/io/gcp/internal/clients/bigquery/bigquery_v2_messages.py#L2194
and more...

Reply:
this is autogenerated from BQs published APIs. This would need to be fixed from BQ side. Feel free to ping me to figure that out.

Reply:
yeah, I was curious to know how they got generated.

Also, are those APIs open-sourced?